### PR TITLE
docs: [tree] unified format

### PR DESCRIPTION
**Pull request by btea**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: docs/en-US/component/tree.md [hunk 1]
```diff
@@ -123,43 +123,43 @@ tree/draggable
...
+| render-after-expand   | whether to render child nodes only after a parent node is expanded for the first time                                                                                                                                                                                                                                                                                       | ^[boolean]                                             | true    |
+| load                  | method for loading subtree data, only works when `lazy` is true                                                                                                                                                                                                                                                                                                             | ^[Function]`(node, resolve, reject)`                   | —       |
```
Comment (tolking) [inline docs/en-US/component/tree.md:133]:
```suggestion
| load                  | method for loading subtree data, only works when `lazy` is true                                                                                                                                                                                                                                                                                                             | ^[Function]`(node, resolve, reject) => void`                   | —       |
```

File: docs/en-US/component/tree.md [hunk 2]
```diff
@@ -123,43 +123,43 @@ tree/draggable
...
+| node-key              | unique identity key name for nodes, its value should be unique across the whole tree                                                                                                                                                                                                                                                                                        | ^[string]                                              | —       |
+| props                 | configuration options, see the following table                                                                                                                                                                                                                                                                                                                              | ^[object]`{children: string, label: string, disabled}` | —       |
```
Comment (tolking) [inline docs/en-US/component/tree.md:131]: The type description here is incomplete. How about link the type to the following props table?

File: docs/en-US/component/tree.md [hunk 3]
```diff
@@ -123,43 +123,43 @@ tree/draggable
...
+| indent                | horizontal indentation of nodes in adjacent levels in pixels                                                                                                                                                                                                                                                                                                                | ^[number]                                              | 18      |
+| icon                  | custom tree node icon component                                                                                                                                                                                                                                                                                                                                             | ^[string] / `Component`                                | —       |
```
Comment (tolking) [inline docs/en-US/component/tree.md:148]:
```suggestion
| icon                  | custom tree node icon component                                                                                                                                                                                                                                                                                                                                             | ^[string] / ^[Component]                               | —       |
```

File: docs/en-US/component/tree.md (hunk 1 above)
Comment (btea) [reply docs/en-US/component/tree.md]: Done.

File: docs/en-US/component/tree.md (hunk 2 above)
Comment (btea) [reply docs/en-US/component/tree.md]: Agree.

File: docs/en-US/component/tree.md (hunk 3 above)
Comment (btea) [reply docs/en-US/component/tree.md]: Done.

**Review by warmthsea — APPROVED** [review]

**Pull request merged**